src/pages/Registration/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
React import is not needed anymore, if you face warnings you can add this command to your eslint config:`"react/react-in-jsx-scope": "off"`